### DISCUSSION TITLE
Timeline per persons

### DISCUSSION THREAD

**Discussion by Kadz93** [immich-app/immich#3004] [Feature Request]
Hi,
I would love to see the timeline option not only on the main page.
It would be super nice to have a timeline if I filter for a specific person, for all persons or specific persons.

With this it would be way easier to give up existing folder structures like
/pictureofdaughter/year1
/pictureofdaughter/year2

Is this possible?

**Comment by jrasm91** [comment]
Yes, there are lots of performance benefits to using the timeline view on multiple pages, but there is some needed internal refactoring before that can happen. #2464 is a WIP to that end.

**Comment by bo0tzz** [comment]
If you open up a person from the explore view, you get a timeline bar.

**Comment by Kadz93** [comment]
I'm not asking for the timeline bar since that Doesn't help me navigate days at all. Im asking for the exact same view as in the home view
